### ISSUE TITLE
Not all coverage files (`*.ec`) included in the JacocoReport when using `clearPackageData`

### ISSUE THREAD

**Issue by EverydayPineapple** [NeoTech-Software/Android-Root-Coverage-Plugin#83]
Our project is setup in a way that outputs seperate `.ec` files per test. I believe this is due to the `clearPackageData` option being set to true, which is desired for us since it helps with test indempotence.

The issue is that the plugin is only looking for `coverage.ec` files when I need it to be looking for `*.ec` files. 

The issue has been discussed [here](https://github.com/NeoTech-Software/Android-Root-Coverage-Plugin/pull/73#issuecomment-1304739778) but I'm making a new issue since I'm not convinced that PR is trying to solve the same thing.

**Comment by Rolf-Smit** [comment]
This is indeed something I never tested with this plugin, I have some more important bugs to work on first such as supporting the Android Gradle Plugin version 8.0. But this is on my radar as an improvement.

**Comment by Rolf-Smit** [comment]
So I added a test case to the plugin (locally not pushed yet) that uses `clearPackageData` however I cannot reproduce the issue.

Do you remember which version of the plugin you where using? Gradle Managed Devices? Test Orchestration?

**Comment by Rolf-Smit** [comment]
Did some more testing and included Test Orchestrator into the mix, now I can reproduce the issue.

I took this configuration as example: https://github.com/android/testing-samples/blob/main/runner/AndroidTestOrchestratorWithTestCoverageSample/app/build.gradle

The bad news, this is not something I can fix as it seems to be a bug in the Android tooling itself:

- https://issuetracker.google.com/issues/126258801
- https://issuetracker.google.com/issues/123987001 (this one is marked as fixed, but I can 100% reproduce the issue on the latest 8.3.0-alpha12 release)

The problem is that the coverage files that this Plugin needs (JaCoCo eventually needs them) are not created or disappear before we can use them.

**Comment by Rolf-Smit** [comment]
Closing this issue for now, as it cannot be fixed for reasons previously described.

**Comment by k4k7us23** [comment]
Looks like, I have managed to fix this issue in https://github.com/NeoTech-Software/Android-Root-Coverage-Plugin/pull/118
